index.js

```diff
@@ -87,17 +89,55 @@ const speakers = [
     quote: 'Member of the Board of Management of BMW AG, Production',
     image: 'assits/speaker3.jpg',
   },
+  {
+    name: 'DR. MILAN NEDELJKOVIĆ',
+    about: 'Production',
+    quote: 'Member of the Board of Management of BMW AG, Production',
+    image: 'assits/speaker3.jpg',
+  },
```

Comment:
- [ ] **Optional**: Hi, According to your project requirement, you should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including the Layout: composition and space between elements.
- Therefore, The Featured Speakers section is supposed to have six individual articles, each with an image. I would strongly suggest that you fix that. This will help you to learn how to copy a given design.
- **See your Page**:
![Capture1](https://user-images.githubusercontent.com/63972670/227231874-fcd872ab-4d3c-49b9-952a-1f4c02df895a.JPG)

- **Original page:**

![Capture1](https://user-images.githubusercontent.com/63972670/227232386-46bff883-4fe7-4812-8154-017fe0e0138e.JPG)